src/appshell/view/appmenumodel.cpp

```diff
@@ -599,7 +599,9 @@ MenuItemList AppMenuModel::makeWorkspacesItems()
     }
 
     items << makeSeparator()
-          << makeMenuItem("configure-workspaces");
+          << makeMenuItem("edit-workspace")
+          << makeMenuItem("configure-workspaces")
+          << makeMenuItem("reset-workspace");
```

Comment:
Maybe it is not a good place for 'edit-workspace' and 'reset-workspace'. 
Because these two options affect to current workspace, 'configure-workspaces' affects all workspaces. 

@Tantacrul 
What do you think about adding these two operations to the Workspaces dialog for each workspace? Like in Parts dialog